easybuild/easyconfigs/s/Spyder/Spyder-5.0.5-foss-2020b.eb

```diff
@@ -0,0 +1,278 @@
+easyblock = 'PythonBundle'
+
+name = 'Spyder'
+version = '5.0.5'
+
+homepage = 'https://github.com/spyder-ide/spyder'
+description = """Spyder is an interactive Python development environment providing MATLAB-like features in a simple
+ and light-weighted software."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'cstd': 'c++11'}
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+dependencies = [
+    ('Python', '3.8.6'),
+    ('ZeroMQ', '4.3.3'),
+    ('PyQt5', '5.15.1'),
+    ('IPython', '7.18.1'),
+    ('geopandas', '0.9.0'),
+    ('libspatialindex', '1.9.3'),
```

Comment:
I really don't think spyder needs these at all. Without it, we can drop this all the way down to `GCCcore`.